### ISSUE TITLE
Using `alembic downgrade`, it's impossible to cancel the initial revision

### ISSUE THREAD

**Issue by vporton** [sqlalchemy/alembic#1326]
**Describe the bug**
Using `alembic downgrade`, it's impossible to cancel the initial revision:

Running `alembic downgrade 2ba30ccd01a7`, does not drop the tables added by the initial migration `2ba30ccd01a7` (despite `downgrade()` script is correct).

**Expected behavior**
Need a way to downgrade to the state before the initial migration (for example, to run the initial migration again after this).

**Error**
No error message.

**Versions.**
 - OS: Ubuntu
 - Python: 3.9.16
 - Alembic: 1.12.0
 - SQLAlchemy: 2.0.22
 - Database: Posgres
 - DBAPI: psycopg2

**Comment by CaselIT** [comment]
Hi,

use `base` instead. the downgrade command reads as `downgrade to the provided revision`.
Admittedly the docs could be clearer, like the command line help does not mention `base` or `head`.

@zzzeek anything against improving upon the docs / command line help?

**Comment by zzzeek** [comment]
Sure, though this is documented?  literally the section called [downgrading](https://alembic.sqlalchemy.org/en/latest/tutorial.html#downgrading) illustrates downgrading to base.

@vporton were the docs not clear here?

**Comment by CaselIT** [comment]
yes, but doing `alembic upgrade -h` or `alembic downgrade -h` makes no mention if it

**Comment by zzzeek** [comment]
"base" is just one of many kinds of identifiers that can be passed, which might not be appropriate in a multi-base environment.  there's also relative revision identifiers, etc.
